test/valgrind/valgrindTestSuite.sh

```diff
@@ -575,13 +614,18 @@ then
         mv /tmp/accumulator_$LISTENER_PORT       $vtest.accumulator_$LISTENER_PORT
         mv /tmp/accumulator_$LISTENER2_PORT      $vtest.accumulator_$LISTENER2_PORT
 
-        failText=$(failedTest "$vtest.valgrind.out" $vtest 0 "valgrind" $xTestNo)
+        failText=$(leakFound "$vtest.valgrind.out" $vtest 0 "valgrind" $xTestNo)
```

Comment:
This means that .vtest are only look for leaks. Is that correct?

(No problem with that, .vtest are not very important at the present time; just to know)